distributed/asyncio.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
```

Comment:
I think that there are two options:

1.  Leave the import here, but make sure that we do not import `distributed.asyncio` in `distributed.__init__`
2.  Import `distributed.asyncio` in `distributed.__init__` and move the `asyncio` import within the class constructors.